tests/staticfiles_tests/test_storage.py

```diff
@@ -31,6 +31,11 @@ class TestHashedFiles(object):
     def tearDown(self):
         # Clear hashed files to avoid side effects among tests.
         storage.staticfiles_storage.hashed_files.clear()
+        # Set max passes back to a sane value in case a test changed it
+        storage.staticfiles_storage.max_post_process_passes = 5
```

Comment:
It seems the original value of the class should be stored by setUp and then restored here in case the default value of 5 changes.